discover/portal/articles/210-setting-up/02-instance-settings/02-instance-settings.markdown

```diff
@@ -0,0 +1,217 @@
+# Instance Settings [](id=instance-settings)
+
+Virtual instance settings are configured in the Instance Settings section of the
+Control Panel. 
+
+Instance Settings has four sections: *Configuration*, *Identification*,
+*Social*, and *Miscellaneous*. This article covers the first and largest
+section, Configuration. The remaining sections are covered in the next article.
+
+![Figure 1: The configuration options in Instance Settings are divided into four categories.](../../../images/instance-settings-navigation.png)
+
+The Configuration section of In stance Settings is divided into seven
+sub-sections. The first is General.
+
+## General [](id=general)
+
+The General section has three headings: Main Configuration, Navigation, and
+Additional Information. Under Main Configuration, set the name of the entity
+responsible for running the virtual instance. Additionally, define the name of
+your virtual instance's default site, set the mail domain, virtual host and
+content delivery network address.
+
+Under the Navigation heading, set a home page for your virtual instance as well
+as default landing and logout pages. To set these pages, use the part
+of the page's address that follows your domain. For example, to set the
+default landing page to be 
+
+`http://localhost:8080/web/guest/login`
+
+use `/web/guest/login`. Use the variable `${liferay:screenName}` as part of the
+address. This comes in handy if you want to redirect Users to their personal
+pages upon login. For example, the User `marvin` has this default URL to his
+personal page: 
+
+    http://localhost:8080/user/marvin
+
+To make sure he's directed there on login, place this in the Default Landing
+Page field:
+
+    /user/${liferay:screenName}
+
+Alternatively, set these URLs in a `portal-ext.properties` file:
```

Comment:
I feel like, if included, these three properties may need more explanation than just saying "Alternatively". 
For example, what if both are set? Which one wins?
Or, what's the difference between a landing page and the home URL?

Reply:
Suggested:

>These URLs can also be set at the system scope, in a `portal-ext.properties`
>file:

>    default.landing.page.path=
>    default.logout.page.path=
>    company.default.home.url=

>All virtual instances will share the values specified in the properties file.
>Changes made in Instance Settings override the values set in the properties
>file.

Reply:
👍